packages/client-react-streaming/src/importErrors.test.tsx

```diff
@@ -1,13 +1,16 @@
 import assert from "node:assert";
 import { test } from "node:test";
-import { outsideOf } from "./util/runInConditions.js";
+import { outsideOf } from "@internal/test-utils/runInConditions.js";
+import { browserEnv } from "@internal/test-utils/react.js";
+import { silenceConsoleErrors } from "@internal/test-utils/console.js";
```

Comment:
Since we now have tests in both bundles, I've moved common test util code into a new package, `@internal/test-utils`.